### DISCUSSION TITLE
smstools3 und umtsd

### DISCUSSION THREAD

**Discussion by pony1k** [Freetz-NG/freetz-ng#307] [Allgemein] [answered]
Ich kann mit smstools3 auf meiner FritzBox 7490 zwar SMSe verschicken, aber leider keine empfangen. Ich möchte den umtsd nicht herauspatchen, weil ich auch das mobile Internet nuzten möchte und das dann ja wahrscheinlich nicht mehr geht. Kann ich die FritzBox irgendwie anders davon abhalten, den SMS-Speicher von meinem UMTS-Stick auszuleeren? Ich habe auch versucht, beim FritzOS-eigenen SMS-Push-Service einen Mailserver im lokalem Netzwerk anzugeben, um so an meine SMS zu kommen. Das resultierte aber leider nur in "TCP-Error".

**Comment by fda77** [comment, marked as answer]
Die Namensauflösung ist bei AVM schon immer total vermurkst. Man kann als DNS keinen lokalen Server angeben. Wenn man es doch macht kann die Fritzbox diesen nicht mehr anpingen, und diese IP kann die Fritzbox nicht mehr anpingen. Ausserdem kann diese IP nicht mehr ins Internet, falls diese Fritzbox der Router ist... Keine Ahnung wie man sowas zusammenknauben kann.
Es hilft leider nur direkt die IP des Mailservers beim Puph-Service einzutragen ^^

Was der umtsd macht kann man von aussen nicht beurteilen. Vielleicht kannst du ihn vom Starten abhalten, indem du duch an diesem Patch orientierst: https://github.com/Freetz-NG/freetz-ng/blob/master/patches/scripts/381-modify_umtsd.sh

Oder gleich ganz removen. Mit 3.G hatte ich dies verwendet, da gab es von AVM noch gar keinen GSM/UMTS-Support: https://freetz-ng.github.io/freetz-ng/make/ppp.html Kann aber sein dass es noch an 4.+5. G angepasst werden muss

**Reply by pony1k** [reply]
Interessant. Es scheint mir, als würde die Fritzbox auch keine lokalen IPs als Mailserver akzeptieren. Gibt auch wieder nur TCP-Error.
Das ppp finde ich interessant. Aber das sieht nach einer Menge Arbeit aus, denn telefonieren möchte ich mit den Stick ja auch noch! Das mach ich jetzt erst mal nicht. Lieber mach ich mir einen Mail-Account irgendwo und hole ich meine SMSe eben dort ab.

**Comment by fda77** [comment]
Lokale IPs gehen auf jeden Fall, das nutze ich so bei allen Fritzboxen die ich so hier hab. Sogar SSL über Port 465.
Telefonieren auch? Ich weiss nicht genau wie das bei den aktuellen Sticks ist, die älteren hatten 1-3 virtuelle TTY-devices zur kommunikation

**Reply by pony1k** [reply]
Lokale IP klappt bei mir jetzt auch, wo der Mailserver richtig konfiguriert ist. Telefonieren funktioniert auch, gerade getestet. Mein Stick macht 4 virtuelle TTY-devives. SMS schicken kann ich auf dem zweiten. Was die anderen genau machen weiß ich nicht.

**Reply by fda77** [reply]
Ich kenn mich mit den neueren Sticks nicht aus, mein neuester ist 3G. Falls es eine Port gibt über das ausschliesslich die SMS abgewickelt werden, könnte man vielleicht udev-Regeln erstellen/ändern so dass dieser nicht vom umtsd genutzt wird
